assets/js/customizer.js

```diff
@@ -0,0 +1,64 @@
+(function() {
+
+	wp.customize.bind( 'ready', function() {
+
+		// An object containing the areas of the theme and their corresponding options for background-colors.
+		// TODO: Move these to PHP and add as vars to the script.
+		var backgroundColors = { // TODO: Define in PHP then add as vars to the script.
+			content: {
+				setting: 'background_color',
+				elements: {
+					text: 'body',
+					accent: 'a, .wp-block-button.is-style-outline, .has-drop-cap:not(:focus):first-letter, a.previous-post, a.next-post'
+				}
+			}
+		};
```

Comment:
Just like in the preview script, these should be moved to a `wp_localize_script` call.